Sources/TuistSupport/System/Systeming.swift

```diff
@@ -53,7 +53,7 @@ public protocol Systeming {
     ///   - environment: Environment that should be used when running the task.
     ///   - redirection: Output Redirection behavior for the underlying `Process`
     /// - Throws: An error if the command fails.
-    func runAndPrint(
+    func run(
```

Comment:
Shouldn't the documentation for this method be updated?

I don't think `Runs a command in the shell printing its output.` is correct as this command should only redirect the output